### ISSUE TITLE
Fortnite News Query

### ISSUE THREAD

**Issue by dpather** [Tustin/fortnite-php#38]
Hi!

Is there a way that I can retrieve the news information located at this link?
https://www.epicgames.com/fortnite/en-US/news

I thought that the code below would get it but it doesn't.

![image](https://user-images.githubusercontent.com/26299862/44001183-54f8afb6-9e2d-11e8-9618-7e0633bbd8cb.png)

Any ideas? Perhaps I am missing something?

Thank you, kindly.

**Comment by Krokit** [comment]
Hi, @dpather i saw that there was api on their blog, but it's not possible to retrieve news from this api. 😄 It's possible to be added over here, but for what will be used? Because this api is made for the game api and personally i think is not for here. 😺

**Comment by dpather** [comment]
Thank you for the reply @Krokit !

I would like to be able to easily display the latest fortnite battleroyale news to players so that they can be kept up to date. I have a news feed section which needs such information.

What do??

**Comment by Krokit** [comment]
I will make some test and if it's working i will push RP. 😄

**Comment by dpather** [comment]
Thank you so much! can't wait to try it! 😄

**Comment by hadouken111** [comment]
Hello, 'Get News' still not working ,any idea ???
Thank

**Comment by atgrau** [comment]
I've solved the problem with **actual news function** in the #42 pull request, waiting for tustin checking this.

**Comment by hadouken111** [comment]
@atgrau thank you you save my day :) but it's normal that only return 3 news ??

**Comment by atgrau** [comment]
@hadouken111 Yes, actually only returns 3 news, here you can see and screenshot of my site showing the actual news:
![image](https://user-images.githubusercontent.com/18336235/46088984-d90fe280-c1ad-11e8-913f-089abf6ab95d.png)

Additionally, you can store the news in your own database to show all the news you want.

**Comment by Krokit** [comment]
In 2/3 days i will push RP for news from the fortnite website. :) @dpather

**Comment by hadouken111** [comment]
this is a great news :dancer:
